crates/macro-support/Cargo.toml

```diff
@@ -17,7 +17,7 @@ extra-traits = ["syn/extra-traits"]
 strict-macro = []
 
 [dependencies]
-syn = { version = '1.0', features = ['visit'] }
+syn = { version = '1.0', features = ['full', 'visit'] }
```

Comment:
Out of curiosity, how come this was added?

Reply:
I included this because the document of `syn` says that [`ExprArray`](https://docs.rs/syn/1.0.26/syn/struct.ExprArray.html) requires the `"full"` feature. But, I now checked that the entire crate seems to work even without this `"full"` feature specification. Does it have something to do with how the `features` is applied for each sub-crate? Anyway I think the dependencies should be included here..

Reply:
Sounds good to me!